src/modules/stakewise/stakewise-daemon/tasks/send-exit-data.go

```diff
@@ -0,0 +1,109 @@
+package swtasks
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/nodeset-org/eth-utils/beacon"
+	"github.com/nodeset-org/hyperdrive/daemon-utils/validator/utils"
+	swcommon "github.com/nodeset-org/hyperdrive/modules/stakewise/stakewise-daemon/common"
+	"github.com/nodeset-org/hyperdrive/shared/utils/log"
+	eth2types "github.com/wealdtech/go-eth2-types/v2"
+)
+
+// Send exit data task
+type SendExitData struct {
+	sp *swcommon.StakewiseServiceProvider
+
+	log log.ColorLogger
+}
+
+// Create Exit data task
+func NewSendExitData(sp *swcommon.StakewiseServiceProvider, logger log.ColorLogger) *SendExitData {
+	return &SendExitData{
+		sp:  sp,
+		log: logger,
+	}
+}
+
+// Update Exit data
+func (t *SendExitData) Run() error {
+	t.log.Println("Checking Nodeset API for Exit data status...")
+
+	// Get services
+	w := t.sp.GetWallet()
+	ns := t.sp.GetNodesetClient()
+	bc := t.sp.GetBeaconClient()
+
+	// Get registered validators
+	resp, err := ns.GetRegisteredValidators()
+	if err != nil {
+		return fmt.Errorf("error getting registered validators: %w", err)
+	}
+
+	// Get validator statuses
+	pubkeys := []beacon.ValidatorPubkey{}
+	for _, v := range resp {
+		pubkeys = append(pubkeys, v.Pubkey)
+	}
+	statuses, err := bc.GetValidatorStatuses(context.Background(), pubkeys, nil)
```

Comment:
We can do a little refactoring here to optimize how the chain is called; this can actually be fairly expensive on the BN so you only want to do it if you know you need a signed exit message, and only on validators that haven't been uploaded yet.

Reply:
Ok let me take care of this